src/index.js

```diff
@@ -0,0 +1,109 @@
+const API_KEY = 'http://localhost:5000';
+
+const increaseTemp = () => {
+    const tempElement = document.querySelector('#tempValue');
+    let currentTemp = Number(tempElement.innerHTML);
+    currentTemp++; 
+    tempElement.innerHTML = currentTemp;
+    setTempColor(currentTemp);
+    setLandscape(currentTemp);
+};
+
+const decreaseTemp = () => {
+    const tempElement = document.querySelector('#tempValue');
+    let currentTemp = Number(tempElement.innerHTML);
+    currentTemp--; 
+    tempElement.innerHTML = currentTemp;
+    setTempColor(currentTemp);
+    setLandscape(currentTemp)
+};
+
+const setLandscape = (currentTemp) => {
+    landscapeElement = document.querySelector('#landscape');
+    
+    if (currentTemp >= 80) {
+        landscapeElement.innerHTML = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    } else if (currentTemp >= 70) {
+        landscapeElement.innerHTML = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (currentTemp >= 60) {
+        landscapeElement.innerHTML = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else {
+        landscapeElement.innerHTML = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    }
```

Comment:
You could create an object that has color data, landscape data, etc for each temperature. Then you could use a loop to go through the object instead of having if/else if/else, which would make the function more concise and look something like:

```js
const TEMP_DATA = {
  90: {
    color: 'red',
    landscape: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'
  }, 
  70: { // other data, etc...}
}

for (const [temp, values] of Object.entries(TEMP_DATA)) { 
  if (currentTemp >= temp) { // this condition needs refactoring, but you get the idea
  state.temp.className = value.color; 
  state.landscape.textContent = value.icon; 
  } 
}
```

Then you can move away from having a brittle implementation that can be tricky to make changes to.